setup.py

```diff
@@ -49,8 +49,10 @@
         "Click<8",
         # "umap-learn",
         "harmonypy>=0.0.5",
-        "bbknn>=1.5.0",
+        "bbknn>=1.5.0,<1.6.0",
         "mnnpy>=0.1.9.5",
+        "scipy<1.9.0",
+        "scikit-learn<1.3.0",
         "scrublet",
         # "scikit-misc",
```

Comment:
```suggestion
```
There are some commented-out dependencies in the install_requires list, which may be unnecessary